content/03-reference/01-tools-and-interfaces/02-prisma-client/01-generating-prisma-client.mdx

```diff
@@ -156,4 +156,4 @@ By generating Prisma Client into `node_modules`, the query engine is kept out of
 
 ## Generating Prisma Client in the `postinstall` hook of `@prisma/client`
 
-The `@prisma/client` package defines its own `postinstall` hook that's being executed whenever the package is being installed. This hook invokes the `prisma generate` command which in turn generates the Prisma Client code into the default location `node_modules/.prisma/client`. Notice that this requires the `prisma` CLI to be available, either as local dependency or as a global installation (it is recommended to always install the `prisma` package as a development dependency, using `npm install prisma --save-dev`, to avoid versioning conflicts though).
+The `@prisma/client` package defines its own `postinstall` hook that's being executed whenever the package is being installed. This hook invokes the `prisma generate` command which in turn generates the Prisma Client code into the default location `node_modules/.prisma/client`. Notice that this requires the `@prisma/cli` to be available, either as local dependency or as a global installation (it is recommended to always install the `@prisma/cli` package as a development dependency, using `npm install @prisma/cli --save-dev`, to avoid versioning conflicts though).
```

Comment:
This might technically not be correct - the CLI binary is called `prisma` which is installed vi the `@prisma/cli` package @mhwelander